pisek/jobs/parts/completeness_check.py

```diff
@@ -0,0 +1,57 @@
+# pisek  - Tool for developing tasks for programming competitions.
+#
+# Copyright (c)   2019 - 2022 Václav Volhejn <vaclav.volhejn@gmail.com>
+# Copyright (c)   2019 - 2022 Jiří Beneš <mail@jiribenes.com>
+# Copyright (c)   2020 - 2022 Michal Töpfer <michal.topfer@gmail.com>
+# Copyright (c)   2022        Jiří Kalvoda <jirikalvoda@kam.mff.cuni.cz>
+# Copyright (c)   2023        Daniel Skýpala <daniel@honza.info>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# any later version.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from pisek.jobs.jobs import Job
+from pisek.jobs.parts.task_job import TaskJobManager, SOLUTION_MAN_CODE
+
+
+class CompletenessCheck(TaskJobManager):
+    """Checks task as a whole."""
```

Comment:
Are you intending for this job to check other things in the future?

Reply:
I have nothing in mind right now, but we don't have other place to do these sort of checks.

Reply:
The name is a little odd, since this job doesn't check the completeness of anything.

Reply:
Completeness of the task?

Reply:
You mean that the task preparation is finished? I guess that makes sense.